<?xml version="1.0"?>
<launch>
     <!-- Robot motor controller -->
    <node pkg="eleven_bringup" type="test_motor_protocal.py" name="motor_controller" output="screen">
        <param name="port" value="/dev/ttyHover"/>
        <param name="baud" value="115200"/>
        
        <param name="cmd_topic" value="cmd_vel"/>

        <param name="wheel_separation" value="0.46"/>
        <param name="wheel_radius" value="0.08255"/>
    </node>
</launch>